packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -377,6 +385,14 @@ export class SpecBuilder {
     return this;
   }
 
+  assertIsShutdown(): this {
```

Comment:
Other "assertions" in this package use the "expect" prefix. Is this sufficiently different that it warrants inconsistency? If so could you document that here?